resources/g2/sprites.json

```diff
@@ -2656,10 +2656,14 @@
         "palette": "keep"
     },
     {
-        "path": "track/intamin/booster_1.png"
+        "path": "track/intamin/booster_1.png",
+        "x_offset": -22,
```

Comment:
Where do these numbers come from? They’re not what’s referenced in the code.

Reply:
I have no idea why, but the co-ords in the drawing code do not match the co-ords in the g2 off-sets.
These will match should flat regular pieces.

Reply:
Have you tested these? The booster sprites are not the same as the regular track sprites, so they could very well need different offsets.